suse_migration_services/units/migrate.py

```diff
@@ -74,6 +74,8 @@ def main():
                     '--auto-agree-with-licenses',
                     '--allow-vendor-change',
                     '--replacefiles',
+                    '--allow-downgrade',
+                    '--no-confirm '
```

Comment:
Agree, with `--no-interactive`, the flag `--no-confirm` is not required

```doc
-y, --no-confirm
     Don’t require user interaction. Alias for the --non-interactive global option
```

Reply:
yeah but does not hurt right ? I really cannot put my hand on fire if --no-interactive prevents all interaction especially if a sub command offers an extra option for it. So my vote is to use both to stay super safe. Thoughts ?

Reply:
I think having the redundancy of the option, i.e. `--non-interactive` and `--no-confirm` is confusing. My $0.02

Reply:
I agree with Robert, I read it and think about it this way: "if the option is there is because it adds/does something that is not done/added by the previous options"

AFAIK, from the documentation, `--no-confirm` is a flag for subcommands while `--no-interactive` means the same but it is global

```
-n, --non-interactive
           Switches to non-interactive mode. In this mode zypper doesn’t ask user to type answers to various prompts,
           but uses default answers automatically. Those default answers also depend on other options like
           --no-gpg-checks or --ignore-unknown.
```

Reply:
ok, will remove it then

Reply:
done